modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheUtils.java

```diff
@@ -2117,6 +2120,29 @@ public static boolean isCacheTemplateName(String cacheName) {
         return strategies;
     }
 
+    /**
+     * Getting available cache entry compression strategies.
+     *
+     * @param kernalCtx Kernal context.
+     * @return Mapping of configuration class to strategy class.
+     */
+    public static Map<Class<? extends EntryCompressionConfiguration>,
+        IgniteClosure<EntryCompressionConfiguration, EntryCompressionStrategy>>
+        entryCompressionStrategies(GridKernalContext kernalCtx) {
+        Map strategies = new HashMap<>();
+
+        // Adding strategies from plugins.
+        EntryCompressionStrategySupplier[] suppliers = kernalCtx.plugins()
+            .extensions(EntryCompressionStrategySupplier.class);
+
+        if (nonNull(suppliers)) {
```

Comment:
Please, don't use `nonNull` in such cases. Most code uses `val != null` and there is no any reason for change this style.
Use `nonNull` only as simple lambda-expressions (see corresponding javadoc).